agimus_controller/agimus_controller/trajectories/sine_wave_configuration_space.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+import pinocchio as pin
+
+from agimus_controller.trajectories.quintic_trajectory import QuinticTrajectory
+from agimus_controller.trajectories.trajectory_base import TrajectoryBase
+from agimus_controller.trajectory import (
+    TrajectoryPoint,
+    TrajectoryPointWeights,
+    WeightedTrajectoryPoint,
+)
+
+
+class SinusWaveConfigurationSpace(TrajectoryBase):
+    """ "Define the trajectory of a sine-wave in configuration Space."""
+
+    def __init__(
+        self,
+        w,
+        scale_duration: np.float64,
+        amp: np.float64,
+        ee_frame_name,
+        w_q,
+        w_qdot,
+        w_qddot,
+        w_robot_effort,
+        w_pose,
+    ):
+        super().__init__(ee_frame_name)
+        self.quint_traj = QuinticTrajectory(scale_duration=scale_duration, amp=amp)
```

Comment:
here amp is a simple scalar I would set it to 1 and then use it to multiply the complete signal.